### PR TITLE
[#13317] Auto merge Dependabot PRs

### LINKED ISSUE #13317: Auto merge Dependabot PRs
### Description

It should be possible to automatically merge Dependabot PRs when all checks pass by utilising GitHub's auto-merge functionality.

### Implementation ideas

_No response_

### PR DESCRIPTION
**Pull request by ryanemerson**
Closes #13317

### CODE REVIEW

**Comment by pruivo** [comment]
@ryanemerson does it need to be backported to other branches?

**Review by pruivo — APPROVED** [review]

File: .github/workflows/dependabot_pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - main
+      - 14.0.x
+      - 15.*.x
+
+    types:
+      - opened
+      - reopened
+
+jobs:
+  createJira:
```
Comment (pruivo) [inline .github/workflows/dependabot_pull_request.yml:15]: may I nitpick about the job name?

Comment (ryanemerson) [reply .github/workflows/dependabot_pull_request.yml]: You may, good spot, fixed!

**Comment by ryanemerson** [comment]
> does it need to be backported to other branches?

Yes, I think it makes sense. I have added the required labels.

**Pull request merged**